packages/client/modules/demo/ClientGraphQLServer.ts

```diff
@@ -245,12 +245,7 @@ class ClientGraphQLServer extends (EventEmitter as GQLDemoEmitter) {
     },
     TaskFooterIntegrateMenuRootQuery: () => {
       return {
-        viewer: {
-          ...this.db.users[0],
-          userOnTeam: {
-            ...this.db.users[0]
-          }
-        }
+        ...this.db.viewer
```

Comment:
-1 This is not the generic viewer, but specific to this particular query. Especially `assigneeTeamMember` and `viewerTeamMember` are renamed properties and should only appear in this query, so just move the values in here directly.